src/Microsoft.Performance.SDK.Runtime/TypeExtensions.cs

```diff
@@ -19,5 +19,17 @@ internal static bool TryGetEmptyPublicConstructor(this Type type, out Constructo
             constructor = null;
             return false;
         }
+
+        internal static bool TryGetEmptyConstructor(this Type type, out ConstructorInfo constructor)
+        {
+            constructor = type.GetConstructor(Type.EmptyTypes);
+            if (constructor != null)
```

Comment:
we can just return constructor != null instead of the if / else, as if the constructor is null, we set it to null and return false anyway.